notebooks/tabular_examples/linear_models/Explaining a model that uses standardized features.ipynb

```diff
@@ -119,6 +111,11 @@
   }
  ],
  "metadata": {
+  "kernelspec": {
```

Comment:
could you please remove this? We want the users to choose the kernel themselves.

Reply:
Sure, I guess this was added automatically by my editor